cardano-cli/src/Cardano/CLI/Output.hs

```diff
@@ -0,0 +1,8 @@
+module Cardano.CLI.Output
+  ( OutputAs(..)
+  ) where
+
+import Data.Eq
+import Text.Show
+
+data OutputAs = OutputAsJson | OutputAsText deriving (Eq, Show)
```

Comment:
This should probably be in  `Cardano.CLI.Shelley.Output`. However I'm also wondering if this is necessary as we use `Maybe OutputFile` to determine if we are rendering as JSON to disk or if we are printing to stdout. Are there instances where we don't render as JSON to disk?

Reply:
I just had this idea that we could have `--json-output=FILE` and if users really want it in `stdout`, they can do this: `--json-output=/dev/stdout`.

Reply:
Fair enough. How about modifying `OutputFile` to encompass this?

Reply:
I thought about this and I think it's better to follow the existing convention.